patches/chromium/render_widget_host_view_base.patch

```diff
@@ -1,73 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Anonymous <anonymous@electronjs.org>
-Date: Thu, 20 Sep 2018 17:46:21 -0700
-Subject: render_widget_host_view_base.patch
-
-... something to do with OSR? and maybe <webview> as well? terrifying.
-
-diff --git a/content/browser/renderer_host/render_widget_host_view_base.cc b/content/browser/renderer_host/render_widget_host_view_base.cc
-index 30ec1b5e2688b94bfe6a9c6cf55a839580e81e68..e9ee463b37ebc61d01fe1ceccbb6a280e33fd268 100644
---- a/content/browser/renderer_host/render_widget_host_view_base.cc
-+++ b/content/browser/renderer_host/render_widget_host_view_base.cc
-@@ -654,6 +654,13 @@ void RenderWidgetHostViewBase::OnFrameTokenChangedForView(
-     host()->DidProcessFrame(frame_token, activation_time);
- }
- 
-+RenderWidgetHostViewBase* RenderWidgetHostViewBase::CreateViewForWidget(
-+    RenderWidgetHost* render_widget_host,
-+    RenderWidgetHost* embedder_render_widget_host,
-+    WebContentsView* web_contents_view) {
-+  return web_contents_view->CreateViewForWidget(render_widget_host);
```

Comment:
Seems like this is still used by OSR https://github.com/electron/electron/blob/a5ec3f7476353336b61ab7d7561beca0ebfdfc64/shell/browser/osr/osr_render_widget_host_view.h#L170

but we could probably inline it rather than as a patch.

Reply:
Yeah I whiffed a spot in search! i'll rework this i think